snippets/base/templates/base/includes/snippet_as.css

```diff
@@ -50,6 +59,11 @@
     display: none;
 }
 
+.dark-theme .snippet .block-snippet-button {
+  background-image: url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgaGVpZ2h0PSIxNiIgdmlld0JveD0iMCAwIDE2IDE2Ij48cGF0aCBmaWxsPSIjRjlGOUZBIiBkPSJNOS40MTQgOGwzLjUzMS0zLjUzMWExIDEgMCAxIDAtMS40MTQtMS40MTRMOCA2LjU4NiA0LjQ2OSAzLjA1NWExIDEgMCAxIDAtMS40MTQgMS40MTRMNi41ODYgOGwtMy41MzEgMy41MzFhMSAxIDAgMSAwIDEuNDE0IDEuNDE0TDggOS40MTRsMy41MzEgMy41MzFhMSAxIDAgMSAwIDEuNDE0LTEuNDE0eiIvPjwvc3ZnPg==");
```

Comment:
@glogiotatidis Fixed... Back to the base64 images. Added a new one here with the right color for dark theme ^